src/main/java/com/github/sgov/server/service/repository/VocabularyRepositoryService.java

```diff
@@ -114,7 +96,7 @@ public List<VocabularyDto> getVocabulariesAsContextDtos(String lang) {
             List<VocabularyDto> contexts = new ArrayList<>();
             final SPARQLRepository repo =
                 new SPARQLRepository(IdnUtils.convertUnicodeUrlToAscii(
-                    repositoryConf.getReleaseSparqlEndpointUrl()));
+                    repositoryConf.getUrl()));
```

Comment:
Not sure why u decided to rewrite getTransitiveImports() to JOPA but not this part.

Reply:
IMHO, we either want to support any SPARQL enpoint that is attached to JOPA or support SPARQL endpoint without JOPA.

Reply:
I was trying to make it more readable, but you are right. I will change getTransitiveImports back to SPARQL endpoint without JOPA.